### PR TITLE
added dashboard and users to database

### PR DESCRIPTION
**Pull request by harmanTheCarMan**
-added dashboard routes, schema, and view
-added user and dashboard seeds
-updated mongoose 4.6.6

### CODE REVIEW

File: models/dashboard.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// grab the things we need
```
Comment (jrobcodes) [inline models/dashboard.js:1]: Remove all comments. Replace all `var` with `const`s

File: models/user.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// grab the things we need
```
Comment (jrobcodes) [inline models/user.js:1]: Remove all comments. Replace all `var` with `const`s

File: routes/index.js [hunk 3]
```diff
@@ -6,4 +6,9 @@ router.get('/', (request, response, next) => {
   response.render('index', { title: 'Dragonboard', logo: 'D' });
 });
 
+/* GET home page. */
+router.get('/dashboard', (request, response, next) => {
```
Comment (jrobcodes) [inline routes/index.js:10]: This should be moved into a dashboard routes file

File: seeds/create_User.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+// if our user.js file is at models/user.js
```
Comment (jrobcodes) [inline seeds/create_User.js:1]: Please remove seed files; we should be adding data through a UI, or API endpoints (which is a good next step here)

File: seeds/create_dashboard.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+// if our user.js file is at models/user.js
```
Comment (jrobcodes) [inline seeds/create_dashboard.js:1]: Please remove seed files; we should be adding data through a UI, or API endpoints (which is a good next step here)

**Review by jrobcodes — CHANGES_REQUESTED** [review]
@harmanlearns @jessehall3

**Comment by jrobcodes** [comment]
Closing since this is an incremental PR

File: models/dashboard.js (hunk 1 above)
Comment (harmanTheCarMan) [reply models/dashboard.js:1]: Fixed all issues per changes requested.

**Pull request closed without merging**